CHANGELOG.md

```diff
@@ -69,6 +71,8 @@ This release contains the [PDV refactor (#4111)](https://github.com/hasura/graph
 - console: allow setting computed fields for views (close #6168) (#6174)
 - cli: add missing global flags for seed command (#5565)
 - cli: allow seeds as alias for seed command (#5693)
+- cli-ext: fix get-shared-modules script execution on mac (#6172)
```

Comment:
Might seem like nitpick but is it possible to make the `cli-ext` fix in a separate PR?